router/router.go

```diff
@@ -200,11 +200,15 @@ func New(cfg *config.Configuration, rateConvertor *currency.RateConverter) (r *R
 	// Metrics engine
 	r.MetricsEngine = metricsConf.NewMetricsEngine(cfg, openrtb_ext.CoreBidderNames(), syncerKeys, moduleStageNames)
 	shutdown, fetcher, ampFetcher, accounts, categoriesFetcher, videoFetcher, storedRespFetcher := storedRequestsConf.NewStoredRequests(cfg, r.MetricsEngine, generalHttpClient, r.Router)
-	// todo(zachbadgett): better shutdown
-	r.Shutdown = shutdown
 
 	analyticsRunner := analyticsBuild.New(&cfg.Analytics)
 
+	// todo(zachbadgett): better shutdown
```

Comment:
Can this TODO be removed?

Reply:
i did not remove this because i do not have context of TODO -  is it okay  to remove this line ? @zachbadgett

Reply:
i will just remove it - i do think there is a better way to shutdown, but i will not do it in this PR